packages/app-bridge/src/AppBridgePlatformApp.ts

```diff
@@ -230,6 +230,18 @@ export class AppBridgePlatformApp implements IAppBridgePlatformApp {
             }
         }
     }
+
+    getAllDocuments(): Promise<Document[]> {
+        return null as any;
+    }
+
+    getDocumentPagesByDocumentId(documentId: number): Promise<DocumentPage[]> {
+        return null as any;
+    }
+
+    getDocumentSectionsByDocumentPageId(documentPageId: number): Promise<DocumentSection[]> {
+        return null as any;
+    }
```

Comment:
Hey @SamuelAlev 
We have some typing issues and to fix some of them i have to add these methods. What do you think would be the best way to adapt these so we could use it for Platform Apps?

Reply:
This shouldn't be added in here, refactor the linkChooser so it displays something like `not supported for plattform apps` or just remove the internal linkSelector button if used with Plattform Apps

Reply:
What does it do?

Reply:
It's basically the linkChooser of a portal for internal links:
![Screenshot 2024-02-08 at 13 30 44](https://github.com/Frontify/brand-sdk/assets/9362990/39a73da4-7b78-4a96-b4d6-99310585dbef)
And then opens this linkTree:
![Screenshot 2024-02-08 at 13 31 15](https://github.com/Frontify/brand-sdk/assets/9362990/7560f76a-9eaf-417f-8dc3-650e49337f56)